### PR TITLE
docs: add `overview` documentation

### PR DESCRIPTION
**Pull request by lawrenceadams**
This pull request includes a significant addition to the `docs/overview.md` file, providing an overview of the `dbt-synthea` project. The changes introduce a detailed description of the project, its structure, and useful resources for further exploration.

Documentation improvements:

* [`docs/overview.md`](diffhunk://#diff-f23678b4c439fd130355644fa4c36770a302e257cc841883c4c11d74918e6e32R1-R33): Added a comprehensive overview of the `dbt-synthea` project, including sections on project structure, exploring the project, and additional resources. Included a project logo for visual enhancement.

### CODE REVIEW

**Comment by lawrenceadams** [comment]
Open to any other ideas @katy-sadowski ! Unsure how much detail to shove in as it then is a repeat of the README.md

File: docs/overview.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{% docs __overview__ %}
+
+## dbt-sythnea
+
+dbt-synthea is a project to demonstrate the power of dbt for building OMOP ETLs. Using [Synthea](https://synthetichealth.github.io/synthea/), it is possible to generate a synthetic patient data which can be transformed into the OMOP standard.
```
Comment (katy-sadowski) [inline docs/overview.md:5]: typo: "synthetic patient dataset"

Comment (katy-sadowski) [inline docs/overview.md:5]: maybe add a hyperlink on dbt as well, to dbt docs?

File: docs/overview.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+### Project Structure
+
+dbt uses parameterized SQL files called [“models”](https://docs.getdbt.com/docs/build/models), which are run in a correct order by building a Directed Acyclic Graph (DAG).
+
+The models have been setup with three distinct stages:
+ - **Staging**: SQL queries that are 1:1 with the source Synthea and vocabulary tables and are used to adjust data types and column names as needed
+ - **Intermediate**: SQL queries used to perform the bulk of the joining and transformation needed to move from source tables into the OMOP CDM, with a focus on joins and transformations which may be reused several times in the final models; for example, source-to-standard concept mappings or modeling of different visit types.
+ - **Mart**: SQL queries that are 1:1 with the final OMOP tables.
+
+### Exploring the Project
```
Comment (katy-sadowski) [inline docs/overview.md:16]: i think we could add all of the original template text here. i realized i didn't know most of the stuff in there after taking a look just now 🙃

File: docs/overview.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+### Project Structure
+
+dbt uses parameterized SQL files called [“models”](https://docs.getdbt.com/docs/build/models), which are run in a correct order by building a Directed Acyclic Graph (DAG).
+
+The models have been setup with three distinct stages:
+ - **Staging**: SQL queries that are 1:1 with the source Synthea and vocabulary tables and are used to adjust data types and column names as needed
+ - **Intermediate**: SQL queries used to perform the bulk of the joining and transformation needed to move from source tables into the OMOP CDM, with a focus on joins and transformations which may be reused several times in the final models; for example, source-to-standard concept mappings or modeling of different visit types.
+ - **Mart**: SQL queries that are 1:1 with the final OMOP tables.
+
```
Comment (katy-sadowski) [inline docs/overview.md:15]: could potentially also add the paragraph about seeds here

File: docs/overview.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+## More Information
+
+ - [What is dbt](https://docs.getdbt.com/docs/introduction)?
+ - [What is OMOP](https://www.ohdsi.org/data-standardization/)?
+ - [What is Synthea](https://synthetichealth.github.io/synthea/)?
+ - [Join OHDSI and collaborate](https://www.ohdsi.org/join-the-journey/)
+ - [Installation](https://github.com/OHDSI/dbt-synthea?tab=readme-ov-file#developer-setup)
+ - [Project Abstract](https://www.ohdsi.org/wp-content/uploads/2024/10/124-Sadowski-dbt-synthea-Abstract-Julien-Nakache.pdf)
+
+![OMOP Logo](assets/OHDSI-logo-with-text-horizontal-colored-white-background.png)
```
Comment (katy-sadowski) [inline docs/overview.md:31]: do you know if it's possible to replace the dbt logo in the upper left with this instead?

**Review by katy-sadowski — CHANGES_REQUESTED** [review]
it looks great!  yay!  thank you!  just a few minor suggestions.

File: docs/overview.md (hunk 4 above)
Comment (lawrenceadams) [reply docs/overview.md:31]: it is with a work around 😉 

image

look good?

File: docs/overview.md (hunk 2 above)
Comment (lawrenceadams) [reply docs/overview.md:16]: sure thing 💪

File: docs/overview.md (hunk 1 above)
Comment (lawrenceadams) [reply docs/overview.md:5]: Good point!

Comment (lawrenceadams) [reply docs/overview.md:5]: Good shout

File: docs/overview.md (hunk 4 above)
Comment (lawrenceadams) [reply docs/overview.md:45]: Implemented with [this](https://github.com/OHDSI/dbt-synthea/pull/105/commits/dd918f6f29a8dd9f951f4903082bf1ccbc4ffc54#diff-fcd4ffb682977fbcdc5e8a679eed82aa4ed2729be1ccb722f002e74be853fb35R47)

File: docs/overview.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
+{% docs __overview__ %}
+
+## dbt-sythnea
```
Comment (katy-sadowski) [inline docs/overview.md:3]: haha, one last typo here in synthea 😆

Comment (lawrenceadams) [reply docs/overview.md:3]: nooo

**Review by katy-sadowski — APPROVED** [review]

**Comment by lawrenceadams** [comment]
Thank you @katy-sadowski ! Sorry for the repeated corrections!... Now to see if it works for real

**Comment by katy-sadowski** [comment]
wohooooo it works and the logo looks great! thank you!!

**Pull request merged**